modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -35,108 +36,144 @@ After the new adapter is ready, the old pod is terminated and all requests are r
+--name=<adapter name>
+--namespace=<adapter namespace>
+--size=<adapter plan size>
+--clientId=<clientId of the application> 
+--clientSecret=<clientSecret of the application> 
+--platformUri=<URL of Anypoint Platform>
 ----
-+	
-Where:
 +
 [%header%autowidth.spread]
 |===
 | Parameter Name | Description | Required/Optional | Default Value
-| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. For the Beta exercise, use “anypoint-service-mesh-adapter” for this parameter. | Required | N/A
-| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. For the Beta exercise, use “marketing” for this parameter | Optional | default
+| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. | Required | N/A
+| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. | Optional | default
 | size | The size of the adapter based on the number of CPUs and memory to use | Optional | Medium
 | clientId | The client ID of the application | Required | N/A
 | clientSecret | The client secret of the application | Required | N/A
 | platformUri | The URL of the host where Anypoint Platform is installed | Optional | https://anypoint.mulesoft.com
 |===
 +
 . Verify the adapter status:
++   
 `./service-mesh adapter list`
 +
 image::provision-service-mesh-adapter-output.png[Example Output]
 +
-. Review adapter logs to see detailed information about Kubernetes events and adapter transactions.
+. Review adapter logs to see detailed information about Kubernetes events and adapter transactions:
++
 `./service-mesh adapter logs --name=<adapter_name> --namespace=<namespace>`
 
+== Create a non-Mule API
+
+If your MuleSoft or non-MuleSoft application does not exist, you must create it before you bind the service to the adapter. 
+
+To create an API, perform the following steps:
+
+. Go to *Anypoint Platform > API Manager* and select the *Manage API* drop-down list. 
+. Select *Create new API*.
++
+The Creating an asset page is displayed.
+. Specify the *Name*, *Asset types* details for the API.  
++
+The Endpoint Confoguration window id displayed, as shown in the following screenshot.
++
+image::create-non-mule-api-service-mesh.png[85%, 85%]
++
+. Specify the required details on the page and click *Save*.
++
+Your API is now created.
 
 == Bind your service to an API on API Manager
-After you set up your namespace, you must bind the service with an existing API to the adapter that you created.
+
+After you provision the adapter, you must bind your service with an existing API to the adapter that you created. The adapter uses the unique API ID of the API for this binding. The Anypoint Platform uses the API ID to identify an application and maintains its configuration. The adapter uses this API ID to fetch policies from the Anypoint Platform and applies to your services that are running in the cluster.
```

Comment:
Please reword "After you provision the adapter, you must bind your service with an existing API to the adapter that you created."

Now you have created your API and provisioned an adapter, you're ready to bind an API with your microservice.